tools/test-proxy/Azure.Sdk.Tools.TestProxy/Admin.cs

```diff
@@ -64,14 +64,21 @@ public async Task AddTransform()
         }
 
         [HttpPost]
-        public async Task RemoveSanitizers([FromBody]RemoveSanitizerList sanitizerList)
+        public async Task RemoveSanitizers()
         {
             DebugLogger.LogAdminRequestDetails(_logger, Request);
+
+            // Originally, this list was parsed using [FromBody], which was implicitly case insensitive. Need to maintain for compat.
+            var sanitizerList = await HttpRequestInteractions.GetBody<RemoveSanitizerList>(Request, new JsonSerializerOptions
+            {
+                PropertyNameCaseInsensitive = true
+            });
+
             var recordingId = RecordingHandler.GetHeader(Request, "x-recording-id", allowNulls: true);
 
             var removedSanitizers = new List<string>();
 
-            if (sanitizerList.Sanitizers.Count == 0)
+            if (sanitizerList.Sanitizers != null && sanitizerList.Sanitizers.Count == 0)
```

Comment:
```suggestion
            if (sanitizerList.Sanitizers == null || sanitizerList.Sanitizers.Count == 0)
```

`default(T)` of `<RemoveSanitizerList>` is class with a single property that may be null. Need to handle empty body here.